packages/fx-core/src/plugins/solution/fx-solution/solution.ts

```diff
@@ -965,9 +966,9 @@ export class TeamsAppSolution implements Solution {
       node.addChild(capNode);
 
       // 1.1 hostType
-      const hostTypeNode = new QTreeNode(FrontendHostTypeQuestion);
-      hostTypeNode.condition = { contains: TabOptionItem.id };
-      capNode.addChild(hostTypeNode);
+      //const hostTypeNode = new QTreeNode(FrontendHostTypeQuestion);
+      //hostTypeNode.condition = { contains: TabOptionItem.id };
+      //capNode.addChild(hostTypeNode);
```

Comment:
Removing the host type parameter needs to change these e2e test cases which uses it:
![image](https://user-images.githubusercontent.com/15262146/140271390-2df1b11e-2899-4f08-89b9-87b4896ca417.png)

Reply:
fixed